README.md

```diff
@@ -1,75 +1,103 @@
-# Determined: Deep Learning Training Platform
-
 <p align="center"><img src="determined-logo.png" alt="Determined AI Logo"></p>
 
-Determined is an open-source deep learning training platform that makes building
-models fast and easy. Determined enables you to:
+Determined is an all-in-one deep learning platform, compatible with PyTorch and Tensorflow.
 
-* **Train models faster** using state-of-the-art distributed training, without
-  changing your model code
-* **Automatically find high-quality models** with advanced hyperparameter tuning
-  from the creators of Hyperband
-* **Get more from your GPUs** with smart scheduling and cut cloud GPU costs by
-  seamlessly using preemptible instances
-* **Track and reproduce your work** with experiment tracking that works
-  out-of-the-box, covering code versions, metrics, checkpoints, and
-  hyperparameters
+It takes care of:
 
-Determined integrates these features into an easy-to-use, high-performance deep
-learning environment — which means you can spend your time building models
-instead of managing infrastructure.
+- **Distributed training**, for faster results.
+- **Hyperparameter tuning**, for obtaining the best models.
+- **Resource management**, for cutting cloud GPU costs.
+- **Experiment tracking**, for analysis and reproducibility.
 
-To use Determined, you can continue using popular DL frameworks such as
-TensorFlow and PyTorch; you just need to update your model code to integrate
-with the Determined API.
 
-## Try out Determined Locally
+# How Determined Works
 
-Follow [these instructions](https://docs.determined.ai/latest/how-to/installation/requirements.html#install-docker) to install and set up docker.
+The main components of Determined are the Python library, the command line interface (CLI), and the Web UI.
 
- ```bash
+## Python Library
+Use the Python library to make your existing PyTorch or Tensorflow code compatible with Determined. 
 
-# Start a Determined cluster locally.
-python3 -m venv ~/.virtualenvs/test
-. ~/.virtualenvs/test/bin/activate
-pip install determined
-# To start a cluster with GPUs, remove `no-gpu` flag.
-det deploy local cluster-up --no-gpu
-# Access web UI at localhost:8080. By default, "determined" user accepts a blank password.
+You can do this by organizing your code into one of the class-based APIs:
+
+```python
+from determined.pytorch import PyTorchTrial
+
+class YourExperiment(PyTorchTrial):
+  def __init__(self, context):
+    ...
+```
+
+Or by using just the functions you want, via the Core API:
+
+```python
+import determined as det
+
+with det.core.init() as core_context:
+    ...
+```
+
+## Command Line Interface (CLI)
 
-# Navigate to a Determined example.
-git clone --recurse-submodules https://github.com/determined-ai/determined
-cd determined/examples/computer_vision/cifar10_pytorch
+Use the CLI to start the Determined cluster locally, or on your favorite cloud service: 
```

Comment:
I imagine local deployment is the most common starter, so should it get priority over `det deploy aws` in the listing?